src/app.jsx

```diff
@@ -192,7 +132,7 @@
     };
     const _pasteItem = (targetPath, asSymlink) => pasteItem(clipboard, targetPath.join("/") + "/", asSymlink, addAlert);
     const _renameItem = () => renameItem(Dialogs, { selected: selectedContext, path, setHistory, setHistoryIndex });
-    const _editPermissions = () => editPermissions(Dialogs, { selected: selectedContext, path });
+    const _editPermissions = () => editPermissions(Dialogs, { selected: selectedContext || rootInfo, path });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-210-20240130-133717-6c7b7601-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)